Libraries/DeviceInfo/RCTDeviceInfoManager.h

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
This is not really a manager. Let's call it `DeviceInfoModule`. I don't get why we have different names for same modules on iOS and Android. cc @nicklockwood